src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,21 @@
+.catalog {
+  padding-top: 50px;
+  margin: 0 auto;
+  display: flex;
+  flex-wrap: wrap;
+  max-width: catalogWidth($cardWidth, $cardMarginR, $cardCount);
+}
+
+@media (min-width: 1024px) {
+  .catalog {
+    max-width: 1024px;
```

Comment:
`max-width: 1200px;` will be enough

Reply:
If  I remove min-width: 1024px; , the test didn't pass